app/src/main/res/values/strings.xml

```diff
@@ -7,6 +7,9 @@
     <string name="action_sign_in_short">Sign in</string>
     <string name="welcome">"Welcome !"</string>
     <string name="invalid_username">Not a valid UWF email</string>
-    <string name="invalid_password">Password must be >5 characters</string>
+    <string name="invalid_password_length">Password must be >8 characters</string>
+    <string name="invalid_password_capitalization">Password must contain at least 1 upper and 1 lower case letter</string>
+    <string name="invalid_password_character">Password must be contain at least 1 special character</string>
```

Comment:
teeny grammar error, delete "be" from the string output for lines 12 and 13.

Reply:
I will edit it tomorrow, good catch